.cargo/config

```diff
@@ -0,0 +1,9 @@
+[alias]
```

Comment:
This file should be named `config.toml` (with extension) according to documentation